piwheels/master/db.py

```diff
@@ -60,7 +60,9 @@
 
 
 ProjectVersionsRow = namedtuple('ProjectVersionsRow', (
-    'version', 'skipped', 'builds_succeeded', 'builds_failed'))
+    'version', 'released', 'build_id', 'skip', 'duration', 'status', 'filename',
+    'filesize', 'filehash', 'builder_abi', 'file_abi_tag', 'platform_tag',
+    'apt_dependencies'))
```

Comment:
OPTIONAL:  Also possible and easier to read and write:
```python
ProjectVersionsRow = namedtuple('ProjectVersionsRow', """
    version released build_id skip duration status filename filesize
    filehash builder_abi file_abi_tag platform_tag apt_dependencies""")
```
    'version', 'skipped', 'builds_succeeded', 'builds_failed'))
    'version', 'released', 'build_id', 'skip', 'duration', 'status', 'filename',
    'filesize', 'filehash', 'builder_abi', 'file_abi_tag', 'platform_tag',
    'apt_dependencies'))